prow/daily-e2e-rbac-no_auth-default.sh

```diff
@@ -26,4 +26,4 @@ set -u
 # Print commands
 set -x
 
-./prow/e2e-suite.sh --default_proxy
+./prow/e2e-suite.sh --default_proxy --cluster_wide --test_vm
```

Comment:
I think we only want --test_vm on the --auth_enable ones. @chxchx what do you think ?